### DISCUSSION TITLE
Q&A site

### DISCUSSION THREAD

**Discussion by guyemerson** [delph-in/docs#115] [General]
The Discourse Q&A site has been down recently.  Do we know what's wrong?

**Comment by olzama** [comment]
I reported that to UW admins a few days ago. Hopefully they will fix it soon!

**Reply by olzama** [reply]
For me the site is still unresponsive but @emilymbender says she can access it. What about you, @guyemerson ? Or others?

**Reply by guyemerson** [reply]
Still not working for me. Sometimes I get a blank page or a server timeout error. Sometimes I get a loading screen that never finishes loading.

**Comment by arademaker** [comment]
It is not working for me.

**Reply by EricZinda** [reply]
Me neither
Correction: worked. It just took a while

**Reply by EricZinda** [reply]
It is pretty unusably slow right now

**Comment by EricZinda** [comment]
Still down... @olzama, any news from UW?

**Reply by goodmami** [reply]
The UW ling dept's IT admin just sent an email that services are back up. There were apparently issues with many services and most have been resolved. The Q&A site seems to be working for me now.

**Comment by olzama** [comment]
The website is up for me! Yay!
